kasa/smart/modules/fan.py

```diff
@@ -20,7 +22,7 @@ def _initialize_features(self):
                 id="fan_speed_level",
                 name="Fan speed level",
                 container=self,
-                attribute_getter="fan_speed_level",
+                attribute_getter=Feature.ANNOTATED_PROPERTY,
```

Comment:
Maybe we should keep the original here, and just always check for annotations during init?

Reply:
I think it's better not to have a bi-directional pointer when it's not needed and to be clear about the fact that this feature get's the property getter from the annotation.

Reply:
Done, now if this isn't passed to the constructor it will look for it in the hints